akka-cluster/src/multi-jvm/scala/akka/cluster/RemoteFeaturesWithClusterSpec.scala

```diff
@@ -27,10 +27,11 @@ class ClusterRemoteFeaturesConfig(artery: Boolean) extends MultiNodeConfig {
   val third = role("third")
 
   private val baseConfig = {
+    val port = if (artery) MultiNodeSpec.udpPort else MultiNodeSpec.tcpPort
```

Comment:
This might be wrong. Note that `artery` is our nickname of the "new remoting". aeron-upd is one of the supported transports of Artery. tcp is another transport of Artery.

Reply:
correct, this test should actually always run on TCP.